src/components/Review/ReviewContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Container, Header, Icon, Label } from 'semantic-ui-react'
+import { useRouter } from '../../utils/hooks/useRouter'
+import { ApplicationDetails } from '../../utils/types'
+import strings from '../../utils/constants'
+
+export interface ReviewContainerProps {
+  application: ApplicationDetails
+}
+
+const ReviewContainer: React.FC<ReviewContainerProps> = ({ application, children }) => {
+  const { replace } = useRouter()
+  const { template, name, org } = application
+  return (
+    <Container>
```

Comment:
Probably want to chuck in an id for this, since it's a fairly major element.

Reply:
Yep, will add that too. Thanks